semgrep_output_v1.atd

```diff
@@ -1230,8 +1249,21 @@ type ci_scan_dependencies = (string * found_dependency list) list
     <python repr="dict">
     <ts repr="map">
 
+
+
+
+(* Response by the backend to the CLI to the POST /results *)
```

Comment:
These changes are unrelated to the change in the title/pr description, can they be broken out into a separate PR?

Reply:
yes ... I've updated the title at least, but this commit is already referenced in semgrep
and used for semgrep-interfaces so I'll merge this one.